capa/features/freeze/__init__.py

```diff
@@ -179,7 +182,7 @@ class BasicBlockFeatures(BaseModel):
 class FunctionFeatures(BaseModel):
     address: Address
     features: Tuple[FunctionFeature, ...]
-    basic_blocks: Tuple[BasicBlockFeatures, ...] = Field(alias="basic block")
+    basic_blocks: Tuple[BasicBlockFeatures, ...] = Field(alias="basic blocks")
```

Comment:
The previous alias appeared incorrect - unsure how this affects code elsewhere as changing to `basic blocks` did not throw any exceptions (pending CI checks).

Reply:
agree its worrying that no tests caught this; however, i think all our json document access goes through the pydantic (de)serializer. so, since reading and writing both would use the wrong property name, it still worked?

Reply:
in theory this is a breaking change, but im not sure its worth being pedantic about that. if someone complains, we can roll back and provide an accessor for both the misspelling and correct property.